### PR TITLE
Update identity-design-principles.md

### PR DESCRIPTION
**Pull request by andresravinet**
Just linking my name to my LinkedIn page.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/solutions/identity-design-principles.md [hunk 1]
```diff
@@ -301,7 +301,7 @@ It is important to understand relationships between various services in the same
...
+Azure Role-Based Access Control (RBAC) enables fine-grained access management for Azure. Using RBAC, you can manage access to resources by granting users the fewest permissions needed to perform their jobs. Details are out of scope for this document, but for more information on RBAC, see [What is role-based access control (RBAC) in Azure?](https://docs.microsoft.com/azure/role-based-access-control/overview) RBAC is important but only part of the governance considerations for Azure. [Cloud Adoption Framework](https://docs.microsoft.com/azure/cloud-adoption-framework/govern/) is a great starting point to learn more. I like how my friend, [Andres Ravinet](https://www.linkedin.com/in/andres-ravinet/) walks customers step-by-step though various components to decide on the approach. High-level view for various elements (not as good as the process to get to actual customer model) is something like this:
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/identity-design-principles.md:304]: Please insert a comma after the LinkedIn link (i.e. "(https://www.linkedin.com/in/andres-ravinet/),) and also correct the typo in the same sentence, i.e. "step-by-step though various components" should be "step-by-step through various components".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@andresravinet Please implement the suggested changes. Thanks,

**Comment by yogkumgit** [comment]
@andresravinet

**Comment by yogkumgit** [comment]
@andresravinet please resolve this conflict so that this PR can be merged. Thanks

**Comment by andresravinet** [comment]
I can't seem to edit the file to resolve the conflicts.

**Comment by tiburd** [comment]
I committed this change to the private repo: https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/10169 
Closing this PR.

**Pull request closed without merging**